test/acceptance/tests/fixtures/cases/static-server-hc/patch.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-server
+spec:
+  template:
+    metadata:
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+    spec:
+      containers:
+        - name: static-server
+          image: kschoche/http-echo:latest
```

Comment:
Did we talk about maybe swapping out this image?

Reply:
had a chat with @lkysow on this topic and we're okay leaving it for now.
If we want to change it for GA I think that is also fine.

Reply:
ship it 😄